api/app.py

```diff
@@ -0,0 +1,43 @@
+# Import your packages 
+from flask import Flask, Response, jsonify, request
+import pickle
+import boto3
+import pandas as pd
+import os
+from snowflake.connector.pandas_tools import write_pandas
+import snowflake.connector
```

Comment:
```suggestion
import pickle
import boto3
import os
import snowflake.connector

import pandas as pd

from snowflake.connector.pandas_tools import write_pandas
from flask import Flask, Response, jsonify, request
```

It is a good practice to group similar imports